server/service/individualServices/ReadInventoryData.js

```diff
@@ -7,97 +7,268 @@ const FIRMWARE = {
+    let ltpAugmentResponse = await IndividualServiceUtility.forwardRequest(consequentOperationClientAndFieldParams, pathParams, requestHeaders, _traceIndicatorIncrementer);
+    if (Object.keys(ltpAugmentResponse).length === 0) {
+      console.log(`${forwardingName} is not success`);
       return new createHttpError.InternalServerError();
     }
-  
-  }
\ No newline at end of file
+
+    equipmentUuidResponse.equipmentUuidList = ltpAugmentResponse[LTP_AUGMENT.MODULE + ":" + LTP_AUGMENT.PAC][LTP_AUGMENT.EQUIPMENT];
+    equipmentUuidResponse.traceIndicatorIncrementer = _traceIndicatorIncrementer;
+
+    return equipmentUuidResponse;
+  } catch (error) {
+    return new createHttpError.InternalServerError(`${forwardingName} is not success with ${error}`);
+  }
+}
+
+/**
+* Prepare attributes and automate the RequestForProvidingAcceptanceDataCausesReadingTheRadioComponentIdentifiers forwarding-construct 
+*    to get equipment information of given mount-name and equipmentuuid.
+* @param {String} mountName Identifier of the device at the Controller
+* @param {Object} equipmentUuidResponse Comprises of EquipmentUuidList and traceIndicatorIncrementer
+* @param {Object} requestHeaders Holds information of the requestHeaders like Xcorrelator , CustomerJourney,User etc.
+* @returns {Object} return equipment uuid list and traceIndicatorIncrementer
+*/
+async function RequestForProvidingAcceptanceDataCausesReadingTheRadioComponentIdentifiers(mountName, equipmentUuidResponse, requestHeaders) {
+  const forwardingName = "RequestForProvidingAcceptanceDataCausesReadingTheRadioComponentIdentifiers";
+  const stringName = "RequestForProvidingAcceptanceDataCausesReadingTheRadioComponentIdentifiers";
+  try {
+    let equipmentInfoList = [];
+    let equipmentInfoResponse = {};
+    let _traceIndicatorIncrementer = equipmentUuidResponse.traceIndicatorIncrementer;
+
+    /****************************************************************************************************
+      * RequestForProvidingAcceptanceDataCausesReadingTheRadioComponentIdentifiers
+      *   MWDI://core-model-1-4:network-control-domain=cache/control-construct={mount-name}
+      *      /equipment={uuid}/actual-equipment
+      *      ?fields=structure(category);manufactured-thing(equipment-type(type-name;part-type-identifier)
+      *            ;equipment-instance(serial-number))
+    *****************************************************************************************************/
+    let equipmentUuidList = equipmentUuidResponse.equipmentUuidList;
+    for (let i = 0; i < equipmentUuidList.length; i++) {
+      let equipmentUuid = equipmentUuidList[i];
+      let pathParams = [];
+      pathParams.push(mountName);
+      pathParams.push(equipmentUuid);
+      let consequentOperationClientAndFieldParams = await IndividualServiceUtility.getConsequentOperationClientAndFieldParams(forwardingName, stringName)
+      _traceIndicatorIncrementer = ++_traceIndicatorIncrementer;
```

Comment:
will _traceIndicatorIncrementer++ work ?

Reply:
Thank you for the comment. Intially was testing with the same but traceIndicator didn't get incremented as expected so changed it to  ++_traceIndicatorIncrementer.

Reply:
Thank you for the comment. Changed the code accordingly.